src/samplers/number.js

```diff
@@ -1,30 +1,63 @@
+const isInteger = (type) => {
+  return type === 'integer';
+};
+
+const DECIMALS = 2;
+const MAX_VALUE = Number.MAX_SAFE_INTEGER;
+const MIN_VALUE = Number.MIN_SAFE_INTEGER;
+
 function getRandomInt(min, max) {
   min = Math.ceil(min);
   max = Math.floor(max);
-  return Math.floor(Math.random() * (max - min + 1)) + min;
+  return Math.floor(Math.random() * (max - min)) + min;
```

Comment:
It returns a number between min (inclusive) and max (exclusive). It can lead to strange behavior.